drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -3722,24 +3836,29 @@ static int mmcsd_probe(FAR struct mmcsd_state_s *priv)
               /* Bit 1: SD version 1.x */
 
               case MMCSD_CARDTYPE_SDV1:
+                finfo("SD version 1.x .\n");
```

Comment:
Will this output make sense with the fall thru?

Reply:
functionality for SD cards was maintained from original implementation, but looking at the code fallthrough should still make sense - the SD initialistion routine should work for both v1.x and 2.x implementations, supporting identification and usage of block mode addressing for SDHC cards.

Reply:
Not when adding print text. A v1 card will say V1 and V2 So which is it?